mmv1/templates/terraform/examples/cloud_run_service_multiple_regions.tf.erb

```diff
@@ -0,0 +1,192 @@
+# Cloud Run service replicated across multiple GCP regions
+
+######################################
+# Terraform (Project-wide resources) #
+######################################
+resource "google_project_service" "compute_api" {
+  project                    = "<%= ctx[:vars]['project_id'] %>"
+  service                    = "compute.googleapis.com"
+  disable_dependent_services = true
+  disable_on_destroy         = false
+}
+
+resource "google_project_service" "run_api" {
+  project                    = "<%= ctx[:vars]['project_id'] %>"
+  service                    = "run.googleapis.com"
+  disable_dependent_services = true
+  disable_on_destroy         = false
+}
+
+# [START cloudloadbalancing_run_multiregion]
+######################################
+# Terraform (Load Balancer creation) #
+######################################
+
+resource "google_compute_global_address" "lb_default" {
+  name    = "<%= ctx[:vars]['myservice_service_ip'] %>"
+  project = "<%= ctx[:vars]['project_id'] %>"
+
+  # Use an explicit depends_on clause to wait until API is enabled
+  depends_on = [
+    google_project_service.compute_api
+  ]
+}
+
+resource "google_compute_backend_service" "lb_default" {
+  name                  = "<%= ctx[:vars]['myservice_backend'] %>"
+  project               = "<%= ctx[:vars]['project_id'] %>"
+  load_balancing_scheme = "EXTERNAL_MANAGED"
+
+  backend {
+    balancing_mode  = "UTILIZATION"
+    capacity_scaler = 0.85 # TODO what does this do? docs are unclear! :(
```

Comment:
🚨

Reply:
Here's some more information on the capacity scaler: https://cloud.google.com/load-balancing/docs/backend-service#capacity_scaler